openapi/sms/v1/instructions.md

```diff
@@ -0,0 +1,244 @@
+
+## Glossary
+
+---
+
+## Setup
+
+### Partner Onboarding
+
+To onboard we require:
+
+- Display name of the partner
+- Base URL for the partner service
+- Supported countries (ideally a list of ISO-3 country codes)
+- Custom Fields for the self-service customer on-boarding
+  - ##### Field properties:
+    | Property        | Description                                            |
+    | --------------- | ------------------------------------------------------ |
+    | Field Name (ID) | Uniquely identifies the field                          |
+    | Type            | Either `text`, `select` or `password`                  |
+    | Label           | Label shown during on-boarding                         |
+    | Allowed Values  | A list of possible values to offer in the select field |
+  - ##### Example custom fields:
+    | ID       | Type       | Label   | Allowed Values  |
+    | -------- | ---------- | ------- | --------------- |
+    | `apiKey` | `password` | API Key | N/A             |
+    | `region` | `select`   | Region  | `Asia`,`Europe` |
+- Features enabled:
+  - Whether your platform supports inbound messages
+
+And we subsequently provide:
+
+- Emarsys SMS Oauth client ID to verify the Emarsys SMS channel requests
+- Callback Oauth client ID and secret to sign callback requests
+
+### Authorization
+
+Requests sent and received by Emarsys will have an OAuth 2 token. This token should be validated by the Partner service.
+
+| URI                                                       | Description                                                      |
+| --------------------------------------------------------- | ---------------------------------------------------------------- |
+| `https://anrjzyscq.accounts400.ondemand.com`              | SAP token issuer host                                            |
+| `https://anrjzyscq.accounts400.ondemand.com/oauth2/token` | SAP IAS access token endpoint                                    |
+| `https://anrjzyscq.accounts400.ondemand.com/oauth2/certs` | SAP IAS JSON Web Key Set (JWKS) certificate information endpoint |
+
+---
+
+## Workflows
+
+Below are the supported workflows of a partner service
+
+| Workflow          | Required            |
+| ----------------- | ------------------- |
+| Client onboarding | <center>✅</center> |
+| Outbound messages | <center>✅</center> |
+| Inbound Messages  | <center>❌</center> |
+
+### Client Onboarding
+
+```mermaid
+sequenceDiagram
+  autonumber
+  participant Client as Client User
+  participant Emarsys as Emarsys SMS
+  participant Partner as Partner Service
+  Note right of Client: Onboarding
+  Client->>Emarsys: Client Configuration
+  Emarsys->>Partner: Client configuration
+  Note right of Partner: Partner stores the configuration used
+
+```
+
+A partner service requires the following endpoints to support client onboarding:
+
+---
+
+- #### [Create/Update Client Config](../partner-service.yaml/paths/~1clients~1{clientId}~1configuration/get)
```

Comment:
```suggestion
- #### [Create/Update Client Config](./partner-service.yaml/paths/~1clients~1{clientId}~1configuration/get)
```

Reply:
we navigate to the parent directory here because in stoplight markdown files are places in a `docs` directory (within the root) whereas the OpenAPI yaml files are directly in the root